index.js

```diff
@@ -1,7 +1,8 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 1", "color: red") 
-
+// var counter = 1;
+// counter = 2;
```

Comment:
This works but for the purpose of this course, we will just focusing on let and const.  Since ES6, let and const are preferred since there are some scoping issues with var.  The answer we were looking for for this was let counter = 1